sdk/mixedreality/Azure.MixedReality.RemoteRendering/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Release History
+
+## 1.0.0-preview.2 (2021-01-19)
+- Tweaks and fixes.
+
+## 1.0.0-preview.1 (2021-01-18)
+- Initial client.
```

Comment:
needs to be filled out - this is from the template

Reply:
There are no changes to report for a library that has not be released.